assets/panel/debugger.properties

```diff
@@ -173,6 +173,10 @@ projectTextSearch.key=CmdOrCtrl+Shift+F
 # modal for searching functions in a file.
 functionSearch.key=CmdOrCtrl+Shift+O
 
+# LOCALIZATION NOTE (gotoLine.key): A key shortcut to jump to
+# a specific line in a file.
+gotoLine.key=CmdOrCtrl+:
```

Comment:
it looks like we have `gotoLineModal.key` is that the same thing?

Reply:
@jasonLaster Looks like there are different keys for Mac and Windows/Linux platform. I use linux and `Ctrl+:` is the key for opening that modal. Should i add `Cmd+Shift+;` also. 😃

Reply:
Hmm, lets look at the code because shift should not be platform dependent